charts/nexus-repository-manager/README.md

```diff
@@ -145,17 +145,17 @@ The following table lists the configurable parameters of the Nexus chart and the
-| `ingress.path`                             | Path for ingress rules. GCP users should set to `/*`                                         | `/`                                                                                                                                             |
+| `ingress.path`                             | Path for ingress rules. GCP users should set to `/*`.                                         | `/`                                                                                                                                             |
 | `tolerations`                              | tolerations list                                                                             | `[]`                                                                                                                                            |
 | `config.enabled`                           | Enable configmap                                                                             | `false`                                                                                                                                         |
 | `config.mountPath`                         | Path to mount the config                                                                     | `/sonatype-nexus-conf`                                                                                                                          |
 | `config.data`                              | Configmap data                                                                               | `nil`                                                                                                                                           |
 | `deployment.annotations`                   | Annotations to enhance deployment configuration                                              | `{}`                                                                                                                                            |
 | `deployment.initContainers`                | Init containers to run before main containers                                                | `nil`                                                                                                                                           |
-| `deployment.postStart.command`             | Command to run after starting the nexus container                                            | `nil`                                                                                                                                           |
+| `deployment.postStart.command`             | Command to run after starting the container                                            | `nil`                                                                                                                                           |
```

Comment:
NOTE: IMO this read better stripped than adding "repository".  Feel free to disagree.